trade-finance-manager-api/src/v1/middleware/validate-sso-feature-flag.ts

```diff
@@ -0,0 +1,39 @@
+import { isTfmSsoFeatureFlagEnabled } from '@ukef/dtfs2-common';
+
+import { HttpStatusCode } from 'axios';
+import { RequestHandler } from 'express';
+
+/**
+ * @private
+ * Validates that the TFM SSO feature flag is in the correct state to access the endpoint
+ * Used internally by validateSsoFeatureFlagIsOn and validateSsoFeatureFlagIsOff
+ */
+const createValidateSsoFeatureFlagMiddleware =
```

Comment:
```suggestion
const validateSsoFeatureFlag =
```

Reply:
This is actually the creator function which returns the validateSsoFeatureFlagMiddleware